test/integration/targets/lookup_sequence/tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```
I don't know any reason to have this in a YAML file, and there are no existing files in the repo that do.

Reply:
Thanks for pointing out that. This was added inadvertently.